docs/firstobject.md

```diff
@@ -105,11 +107,45 @@ We will come back to this later.
 </div>
 </details>
 
-Make sure you select `Photograph` as `Media Type` and all the fields with a red `*` are filled up. Press `Next: Linked data`. That will load the next step inline.
+Make sure you select `Photograph` as `Media Type` and all the fields with a red `*` are filled up. Press `Next: Linked data`. That will load the next step inline. 
 
 Note: Don't press `Save`, Don't press `Preview`. Why? You are not ready yet!
 
+### Linked Data
+
 ![Ingest Step2](../imgs/ingest-step2.jpg)
 
-This Step showcases some of the Autocomplete Linked data Webform elements we built for Archipelago. We truly believe in Wikidata as a open, honest, source of Linked Open Data and also one where you can contribute back. But we also have LoC autocompletes and Getty. More to come!
+As the name of this step suggests; on this page you will be adding all your Linked Data for your Digital Object. This Step showcases some of the Autocomplete Linked data Webform elements we built for Archipelago. We truly believe in Wikidata as an open, honest, source of Linked Open Data and also one where you can contribute back. But we also have LoC autocompletes and Getty.
+
+Click: `Next: Collections`
+
+### Collections
+
+![Ingest Step3](../imgs/inges-step-3_Collections.jpg)
+
+Since this is our first digital object we do not yet have a Digital Object *Collection* for which `My First Digital Object` could be a member of. In other words, you can leave `Collection Membership` blank and click `Next: Upload Files`.
+
+### Upload Files
+
+![Ingest Step4](../imgs/ingest-step-4_Upload-files.jpg)
+
+This step is pretty straight forward: Click `Choose Files` to open your computer's directory and select which file you would like to ingest. After you've done this, click `Save Metadata`
+
+From here, Steps 5 & 6 (hotspots & Preview) are skipped and you are directed to the **Complete** step.
+
+### Complete
+
+![Ingest Step7](../imgs/ingest-step-7_Complete.jpg)
+
+You made it! You added your Metadata, Linked Data, Uploaded your files and now you're ready to save! First, give your object a preivew by clicking the `Preview` button.
```

Comment:
typo preivew = preview. Since there are two preview buttons, i would suggest to be explicit about which button people should press. Also, we should explain people that "saving metadata" just persist metadata in the current  webform session, but the actual Object ingest happens when the user presses "save"

Reply:
Great. Solved